### DISCUSSION TITLE
科研資源是否有絕對優先度

### DISCUSSION THREAD

**Discussion by bookbug666** [LmeSzinc/AzurLaneAutoScript#2064] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/29662367/210298090-3ec020b9-62cc-4b0e-92c9-bac17f40fe15.png)
同時存在兩個僅0.5到底是誰會更優先呢?

**Comment by ec7529a2** [comment, marked as answer]
优先级以过滤器为准，这三个下拉选项只是附加条件

**Comment by LmeSzinc** [comment]
另外统计显示只会同时出现一个 0.5 科研，你这里科研不用物资的话那基本就是没收益的了
